app/src/lib/editors/win32.ts

```diff
@@ -109,12 +109,16 @@ const editors: IWindowsExternalEditor[] = [
       CurrentUserUninstallKey('{771FD6B0-FA20-440A-A002-3B3BAC16DC50}_is1'),
       // 32-bit version of VSCode (user)
       CurrentUserUninstallKey('{D628A17A-9713-46BF-8D57-E671B46A741E}_is1'),
+      // ARM64 version of VSCode (user)
+      CurrentUserUninstallKey('{D9E514E7-1A56-452D-9337-2990C0DC4310}_is1'),
       // 64-bit version of VSCode (system) - was default before user scope installation
       LocalMachineUninstallKey('{EA457B21-F73E-494C-ACAB-524FDE069978}_is1'),
       // 32-bit version of VSCode (system)
       Wow64LocalMachineUninstallKey(
         '{F8A2A208-72B3-4D61-95FC-8A65D340689B}_is1'
       ),
+      // ARM64 version fo VSCode (system)
```

Comment:
Tiny typo in this comment:
```suggestion
      // ARM64 version of VSCode (system)
```

Reply:
Good catch! Thanks :)